socorro/external/es/super_search_fields.py

```diff
@@ -252,190 +243,6 @@ def add_doc_values(value):
             add_doc_values(field)
 
 
-class SuperSearchFieldsData:
-    """Data class for super search fields.
-
-    This just holds the FIELDS and some accessors to get them.
-
-    """
-
-    def get_fields(self):
-        """Return all the fields from our super_search_fields.json file."""
-        return FIELDS
-
-    # Alias ``get`` as ``get_fields`` so this can be used in the API well and
-    # can be conveniently subclassed by ``SuperSearchFields``.
-    get = get_fields
-
-
-@dataclass
-class IndexDataItem:
-    name: str
-    start_date: datetime.datetime
-    count: int
-
-
-class SuperSearchFields(SuperSearchFieldsData):
```

Comment:
This did three things:

1. set up the webapp for getting `FIELDS`: I changed the webapp so it has its own `SUPERSEARCH_FIELDS` and accesses that directly.
2. contain the code for `test_mapping`: This is vestigial code that we don't use anymore, so I removed it entirely.
3. contain the code for supersearch status: This is only used by the admin in the webapp, so I moved all that code there.

Given that, there's no reason to have this code in this module.